Nodejs/Product/Nodejs/Templates/Files/TypeScriptTsConfig/jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Why are you putting these files in the same folder with the tsconfig.json template files, when every other file gets its own folder? Separate into a Templates/Files/JsConfig folder.

Reply:
I thought of not cluttering the with a lot of folders as both of them are similar files. But I don't have a preference in one or another. Changed.